### PR TITLE
[ticket/14742-32x] Further improvements to migrator

### PR DESCRIPTION
**Pull request by Elsensee**
**This is the PR for 3.2.x. The PR for 3.1 is #4419**

This Pull Request fixes some things that were forgotten in the last PRs (#4409 and #4410) since they were merged after some miscommunication.

Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [x] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket:

https://tracker.phpbb.com/browse/PHPBB3-14742

PHPBB3-14742

### CODE REVIEW

File: phpBB/phpbb/console/command/db/console_migrator_output_handler.php [hunk 1]
```diff
@@ -54,11 +54,11 @@ public function write($message, $verbosity)
 		{
 			$translated_message = call_user_func_array(array($this->user, 'lang'), $message);
 
-			if ($verbosity === migrator_output_handler_interface::VERBOSITY_NORMAL)
+			if ($verbosity === OutputInterface::VERBOSITY_NORMAL)
```
Comment (CHItA) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: what's the reason for all these changes?

Comment (Elsensee) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: Because the output handler for the CLI worked with symfony constants, the migrator however did with ours.
And since they would otherwise be the same I changed all of them to use Symfony constants.
Furthermore I changed ours to be like the Symfony constants for BC reasons. I suggest to remove them in the future, though.

Comment (CHItA) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: then i would say let's revert these changes.

Comment (Elsensee) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: So you'd say, just change the values of the constants we use?

Comment (CHItA) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: that or just map the ones in the console migrator output handler, or whatever. it makes no difference, and it is mostly just a question of style, but i don't see why it would be reasonable to include a class just for its constants. also the symfony console is designed for the console, while the output handler is not, and therefore their similarity should be treated as a coincidence imho (meaning only be used in the console output handler).

Comment (Elsensee) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: Okay, then I'll go with changing the constants' values.

Comment (Elsensee) [inline phpBB/phpbb/console/command/db/console_migrator_output_handler.php]: Fixed now.
